cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -147,11 +147,13 @@ var (
 func contactInfoCommands(entity string, getID func(cmd *cobra.Command, args []string) (*ttnpb.EntityIdentifiers, error)) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "contact-info",
-		Short: fmt.Sprintf("Manage %s contact info", entity),
+		Short: fmt.Sprintf("Manage %[1]s contact info (DEPRECATED. Instead, use administrative_contact and technical_contact fields of %[1]s)", entity),
```

Comment:
Why isn't this one `Hidden` too ?

Reply:
Because for now we still need it for user email address validation